projects.go

```diff
@@ -347,6 +349,9 @@ type CreateProjectOptions struct {
 	OnlyAllowMergeIfAllDiscussionsAreResolved *bool            `url:"only_allow_merge_if_all_discussions_are_resolved,omitempty" json:"only_allow_merge_if_all_discussions_are_resolved,omitempty"`
 	LFSEnabled                                *bool            `url:"lfs_enabled,omitempty" json:"lfs_enabled,omitempty"`
 	RequestAccessEnabled                      *bool            `url:"request_access_enabled,omitempty" json:"request_access_enabled,omitempty"`
+	TagList                                   *[]string        `url:"tag_list,omitempty" json:"tag_list,omitempty"`
```

Comment:
Did you test if this works as expected? As I know that labels are also an Array, but still need some special handling. So I made a specific type for that. Not sure if that is also the case for tags?

Reply:
`*[]string` works for `TagList` - tested it with my CLI:

    golab project create --name NAME --namespace_id ID --tag_list tag1,tag2,tag3

resulted in

      "tag_list": [
        "tag1",
        "tag2",
        "tag3"
      ],

in the created project